src/Xamarin.Android.Build.Tasks/Tasks/GetAotArguments.cs

```diff
@@ -0,0 +1,253 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Android.Build.Tasks;
+using Microsoft.Build.Framework;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+	/// <summary>
+	/// The <Aot/> task subclasses this in "legacy" Xamarin.Android.
+	/// Called directly in .NET 5 to populate %(AotArguments) metadata.
+	/// </summary>
+	public class GetAotArguments : AndroidAsyncTask
```

Comment:
This looks redundant, it should leverage AOT tasks from dotnet/runtime

Reply:
This is used to populate `%(AotArguments)`, which is set on every assembly. It looks like it is the only way pass in paths to the Android NDK, etc. such as:

```xml
<AotArguments>tool-prefix=/some/path;mtriple=arm64-v8a</AotArguments>
```

This is reusing a small amount of code from our current `<Aot/>` task for this.